senders/telegram/send.go

```diff
@@ -50,7 +50,7 @@ func (sender *Sender) buildMessage(events moira.NotificationEvents, trigger moir
 	tags := trigger.GetTags()
 	emoji := emojiStates[state]
 
-	title := fmt.Sprintf("%s%s %s %s (%d)\n", emoji, state, trigger.Name, tags, len(events))
+	title := fmt.Sprintf("%s%s %s %s (%d)\n%s\n", emoji, state, trigger.Name, tags, len(events), trigger.Desc)
```

Comment:
This is not going to work. Any description longer than ~500-600 characters will silently break Telegram notifications due to message length limit (see lines 22..24). You need to cut description according to the limits.

Reply:
You also need to convert the description to a format that supports telegram, as is done, for example, in slack (there we use the blackfriday-slack package), there’s no such library for a telegram and we need to implement on your own

Reply:
We can tolerate bad markup for now. But length limit is a show-stopper: if we merge this PR as-is, some users will silently lose their alerts.

Reply:
@beevee I fixed it.